doc/developers/contributing.rst

```diff
@@ -736,7 +736,7 @@ To also generate the example gallery you can use:
 
 .. prompt:: bash $
 
-    make html
+    make doc
```

Comment:
I don't think this is correct for the `Makefile` of the `/doc` folder. This is only if you build the doc from the root directory. Above we state that you should be in the `/doc` folder.

Reply:
I read too fast. But I never go into that path, I always use make doc:smirk: